src/common/interop/LayoutMapManaged.idl

```diff
@@ -6,7 +6,7 @@ namespace PowerToys
             LayoutMapManaged();
             String GetKeyName(UInt32 key);
             UInt32 GetKeyValue(String name);
-            void Updatelayout();
+            void UpdateLayout();
         }
     }
 }
```

Comment:
I try not to touch files directly, but an editor is likely to want to put in the missing newline at eof (and I'd support it in principle)